x/cwfees/types/cw.go

```diff
@@ -0,0 +1,83 @@
+package types
+
+import (
+	"fmt"
+
+	"github.com/CosmWasm/wasmd/x/wasm/types"
+	wasmVmTypes "github.com/CosmWasm/wasmvm/types"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/gogoproto/proto"
+)
+
+// SudoMsg defines the message sudo enum that is sent to the CosmWasm contract.
+type SudoMsg struct {
+	// CWGrant defines the enum variant of the grant message.
+	CWGrant *CWGrant `json:"cw_grant"`
```

Comment:
CWGrant to CWFees?

Reply:
I think CWGrant is fine here, whilst the "module name" is CWFees (as it handles fees); what is being sent as request here is a request for a fee grant. 

The name is kept as short as possible to avoid extra gas consumption when we traverse the wasm boundary.